src/mpid/ch3/src/mpid_init.c

```diff
@@ -86,7 +86,16 @@ static int set_eager_threshold(MPIR_Comm *comm_ptr, MPIR_Info *info, void *state
 }
 
 
-int MPID_Init(int *argc, char ***argv, int requested, int *provided)
+int MPID_Init_thread_level(int *argc, char ***argv, int requested, int *provided)
```

Comment:
Adding a new ADI function is not a refactor. This change should be split out and reviewed on its own.